r-tidyverse.qmd

```diff
@@ -1,45 +1,110 @@
+```
 
 ## The working environment
 
-### R, RStudio and the tidyverse
+### R, RStudio, the tidyverse and penguins
 
-- R is a fully featured programming language, but it excels as an environment for (statistical) data analysis ([https://www.r-project.org](https://www.r-project.org))
+- R is a fully featured programming language, but it excels as an environment for (statistical) data analysis (<https://www.r-project.org>)
 
-- RStudio is an integrated development environment (IDE) for R (and other languages): ([https://www.rstudio.com/products/rstudio](https://www.rstudio.com/products/rstudio))
+- RStudio is an integrated development environment (IDE) for R (and other languages) (<https://www.rstudio.com/products/rstudio>)
 
-- The tidyverse is a collection of R packages with well-designed and consistent interfaces for the main steps of data analysis: loading, transforming and plotting data ([https://www.tidyverse.org](https://www.tidyverse.org))
-  - This introduction works with tidyverse ~v1.3.0
-  - We will learn about `readr`, `tibble`, `ggplot2`, `dplyr`, `magrittr` and `tidyr`
+- The tidyverse is a powerful collection of R packages with well-designed and consistent interfaces for the main steps of data analysis: loading, transforming and plotting data (<https://www.tidyverse.org>)
+  - This tutorial works with tidyverse ~v2.0
+  - We will learn about the packages `readr`, `tibble`, `ggplot2`, `dplyr`, `magrittr` and `tidyr`
   - `forcats` will be briefly mentioned
   - `purrr` and `stringr` are left out
 
+- The `palmerpenguins` package provides a neat example dataset to learn data exploration and visualization in R (<https://allisonhorst.github.io/palmerpenguins>)
+
+### Preparing the environment for this tutorial
+
+- This tutorial was prepared for the SPAAM Summerschool 2023, where participants work on pre-configured virtual machines and do not have to install any additional software. The instructors will show in the beginning how to create an RStudio project and how to access the training data.
+- If you're reading this later and want to tinker with the example code, then you will need to perform the following steps first:
+  - Install R and RStudio for your operating system according to one of the many instructions online (e.g. [here](https://www.r-bloggers.com/2022/01/how-to-install-and-update-r-and-rstudio))
+  - Download the .qmd file underlying this webpage [here](https://github.com/SPAAM-community/intro-to-ancient-metagenomics-book/blob/main/r-tidyverse.qmd) and copy it to a new directory
+  - Open RStudio and create a new project in this directory with `File` -> `New Project...` -> `Existing directory`
+  - Open this very file from the RStudio project, so that you can easily follow along
+  - Run the `Environment preparation code` code to install necessary R package dependencies and prepare the required data used below
+  
+<details>
+<summary>Environment preparation code</summary>
```

Comment:
I will replace this with a collapsable callout, after it's merged (for consistency with others)